plugin/src/main/java/com/denizenscript/denizen/events/block/BrewsScriptEvent.java

```diff
@@ -1,12 +1,17 @@
 package com.denizenscript.denizen.events.block;
 
 import com.denizenscript.denizen.objects.InventoryTag;
+import com.denizenscript.denizen.objects.ItemTag;
 import com.denizenscript.denizen.objects.LocationTag;
 import com.denizenscript.denizen.events.BukkitScriptEvent;
 import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.Material;

```

Comment:
You seem to have redundant imports now - can remove these manually or use the IDE's automatic import cleanup (`Ctrl + Alt + O`)

Reply:
i never knew that shortcut existed, thanks aya!